kornia/augmentation/functional/__temp__.py

```diff
@@ -1,24 +0,0 @@
-from typing import Callable
```

Comment:
This can be reused later. I think that what @cceyda meant in the previous comment.